docker/Dockerfile

```diff
@@ -1,48 +1,15 @@
-FROM ubuntu:22.04
+ARG BASE_IMAGE=ubuntu:22.04
+
+FROM $BASE_IMAGE
 
 ARG MASTER
```

Comment:
master seems not reproduced, can you pass the argument to the script?

Reply:
Fixed